client/my-sites/site-settings/disconnect-site/down-flow.jsx

```diff
@@ -57,7 +57,7 @@ function DownFlow( { confirmHref, backHref, site, recordTracksEvent: tracks } )
 					<FormattedHeader
 						isSecondary
 						align="left"
-						headerText={ translate( 'Confirm that your site loads.' ) }
+						headerText={ translate( 'Confirm that your site loads' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.